### DISCUSSION TITLE
I was emailed and told I had access to the CLI, but it says unauthorized when I try to auth

### DISCUSSION THREAD

**Discussion by jasonjmcghee** [community/community#52745] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

As described in the subject. GitHub support says no questions about technical previews so posting here.

**Comment by Akash1134** [comment, marked as answer]
Hi @jasonjmcghee ,

An active subscription to GitHub Copilot for Business or GitHub Copilot for Individuals is required for access to GitHub Copilot for CLI. Could you please [check that you have an active subscription](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/viewing-your-subscriptions#reviewing-all-of-your-subscriptions) ? Thanks!

**Reply by jasonjmcghee** [reply]
I do not have an active subscription.

It might be worth including that detail in the email and in the error message.

**Reply by mattrothenberg** [reply]
Thanks for the feedback @jasonjmcghee ! We'll update the error message copy to be clearer.
